sdks/python/apache_beam/pipeline_test.py

```diff
@@ -405,6 +405,12 @@ def process(self, element, timestamp=DoFn.TimestampParam):
     assert_that(pcoll, equal_to([MIN_TIMESTAMP, MIN_TIMESTAMP]))
     pipeline.run()
 
+  def test_timestamp_param_map(self):
```

Comment:
I don't think this PR fixes FlatMap, since it was already working?
I undid your change in core.py and wrote a test for FlatMap and it worked:
```py
   def test_timestamp_param_flatmap(self):
    with TestPipeline() as p:
      assert_that(
          p | Create([1, 2]) | beam.FlatMap(lambda _, t=DoFn.TimestampParam: [t]),
          equal_to([MIN_TIMESTAMP, MIN_TIMESTAMP]))
```